diff -up akonadiconsole-20.03.90/src/jobtrackersearchwidget.cpp.omv~ akonadiconsole-20.03.90/src/jobtrackersearchwidget.cpp
--- akonadiconsole-20.03.90/src/jobtrackersearchwidget.cpp.omv~	2020-04-13 01:50:08.233505306 +0200
+++ akonadiconsole-20.03.90/src/jobtrackersearchwidget.cpp	2020-04-13 01:50:12.808556442 +0200
@@ -53,11 +53,7 @@ JobTrackerSearchWidget::JobTrackerSearch
     mSelectColumn->addItem(QStringLiteral("Job Type"), 4);
     mSelectColumn->addItem(QStringLiteral("State"), 5);
     mSelectColumn->addItem(QStringLiteral("Info"), 6);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(mSelectColumn, QOverload<int>::of(&QComboBox::currentIndexChanged), this, &JobTrackerSearchWidget::slotColumnChanged);
-#else
-    connect(mSelectColumn, QOverload<int, const QString &>::of(&QComboBox::currentIndexChanged), this, &JobTrackerSearchWidget::slotColumnChanged);
-#endif
 }
 
 JobTrackerSearchWidget::~JobTrackerSearchWidget()
diff -up akonadiconsole-20.03.90/src/searchwidget.cpp.omv~ akonadiconsole-20.03.90/src/searchwidget.cpp
--- akonadiconsole-20.03.90/src/searchwidget.cpp.omv~	2020-04-13 01:49:05.217800981 +0200
+++ akonadiconsole-20.03.90/src/searchwidget.cpp	2020-04-13 01:49:09.244845992 +0200
@@ -117,11 +117,7 @@ SearchWidget::SearchWidget(QWidget *pare
 
     connect(button, &QPushButton::clicked, this, &SearchWidget::search);
     connect(mDatabaseView, &QListView::activated, this, &SearchWidget::fetchItem);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(mStoreCombo, QOverload<int>::of(&KComboBox::currentIndexChanged), this, &SearchWidget::openStore);
-#else
-    connect(mStoreCombo, QOverload<int, const QString &>::of(&KComboBox::currentIndexChanged), this, &SearchWidget::openStore);
-#endif
 
     openStore(0);
 
